contrib/packaging/rpm/skydive.te

```diff
@@ -0,0 +1,99 @@
+policy_module(skydive, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type skydive_t;
+type skydive_exec_t;
+init_daemon_domain(skydive_t, skydive_exec_t)
+
+permissive skydive_t;
+
+type skydive_var_lib_t;
+files_type(skydive_var_lib_t)
+
+########################################
+#
+# skydive local policy
+#
+allow skydive_t self:fifo_file rw_fifo_file_perms;
+allow skydive_t self:unix_stream_socket create_stream_socket_perms;
+
+manage_dirs_pattern(skydive_t, skydive_var_lib_t, skydive_var_lib_t)
+manage_files_pattern(skydive_t, skydive_var_lib_t, skydive_var_lib_t)
+manage_lnk_files_pattern(skydive_t, skydive_var_lib_t, skydive_var_lib_t)
+files_var_lib_filetrans(skydive_t, skydive_var_lib_t, { dir file lnk_file })
+
+domain_use_interactive_fds(skydive_t)
+
+files_read_etc_files(skydive_t)
+
+auth_use_nsswitch(skydive_t)
+
+miscfiles_read_localization(skydive_t)
+
+sysnet_dns_name_resolve(skydive_t)
+
+require {
+	type container_var_run_t;
+	type unconfined_t;
+	type skydive_var_lib_t;
+	type container_t;
+	type skydive_exec_t;
+	type skydive_t;
+	type devlog_t;
+	type openvswitch_t;
+	type openvswitch_var_run_t;
+	class system module_load;
+	class file { getattr map open read };
+	class capability { dac_override dac_read_search net_raw sys_admin };
+	class packet_socket { bind create getopt ioctl map setopt };
+	class unix_dgram_socket ioctl;
+	class netlink_route_socket nlmsg_write;
+	class dir { add_name read remove_name search write };
+	class lnk_file read;
+	class sock_file { create unlink write };
+}
+
+#============= openvswitch_t ==============
+allow openvswitch_t self:system module_load;
+files_map_kernel_modules(openvswitch_t)
+
+#============= skydive_t ==============
+allow skydive_t container_t:dir search;
+allow skydive_t container_t:file read;
+allow skydive_t container_t:lnk_file read;
+allow skydive_t container_var_run_t:dir read;
+allow skydive_t devlog_t:lnk_file read;
+allow skydive_t devlog_t:sock_file write;
+allow skydive_t openvswitch_var_run_t:dir { add_name remove_name write };
+allow skydive_t openvswitch_var_run_t:sock_file { create unlink };
+allow skydive_t self:capability { dac_override dac_read_search net_raw sys_admin };
+allow skydive_t self:netlink_route_socket nlmsg_write;
+allow skydive_t self:packet_socket { bind create getopt ioctl map setopt };
+allow skydive_t self:unix_dgram_socket ioctl;
+allow skydive_t skydive_exec_t:file map;
+allow skydive_t skydive_var_lib_t:file map;
+allow skydive_t unconfined_t:file { getattr open read };
```

Comment:
this shouldn't happen, what's the avc statement for this unconfined file access?